modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_het_hom.md

```diff
@@ -0,0 +1,128 @@
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-rmp
+         caption=ABTR RGMB Reactor Mesh Parameters example.
+         block=Mesh/rmp
+
+## Pin structures using PinMeshGenerator
+
+[PinMeshGenerator.md] defines the constituent pin structures used for stitching into assemblies. The pin pitch, number of azimuthal sectors, and geometry / region ID information about each ring, background, and duct region are specified here.
+
+[PinMeshGenerator.md] is called multiple times to define the various pin structures (3 fuel pin types and 1 control pin type).
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-pin
+         caption=ABTR RGMB pin example.
+         block=Mesh/fuel_pin_1
+
+Tips
+
+- Use a unique [PinMeshGenerator.md] block for each pin with a unique geometrical configuration and/or region ID composition
+- region_ids is a 2-dimensional array containing region IDs (essentially materials). The first row of the array represents the 2D radial regions (from center of the pin to outermost region) for the bottom layer of the pin. Each subsequent row assigns IDs on another axial level, from bottom to top. In this case, each pin has 3 radial regions (fuel, clad, background), this array is a column pertaining to each axial level of the pin assembly.
+- While the mesh is still 2D during this step, the axially dependent region IDs are stored for later use during the extrusion step.
+
+## Assembly structures using AssemblyMeshGenerator
+
+[AssemblyMeshGenerator.md] takes the pin types previously defined and places them into a regular hexagonal grid. Additionally, coolant and duct regions need to be added around the pins in order to create the assembly geometry.
+
+[AssemblyMeshGenerator.md] is called multiple times to define the various heterogeneous assemblies (3 fuel assemblies and 1 control assembly).
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-het-assembly
+         caption=ABTR RGMB heterogeneous assembly example.
+         block=Mesh/fuel_assembly_1
+
+Tips
+
+- Use a unique [AssemblyMeshGenerator.md] block for each assembly with a unique geometrical configuration, region ID composition, and/or inventory of pin structures
+- background_region_ids is a 1-dimensional array containing region IDs for each axial layer of the background region. duct_region_ids is a 2-dimensional array containing regions IDs for the duct region. The first row of the array represents the 2D duct regions (from innermost to outermost duct region) for the bottom layer of the assembly. Each subsequent row assigns IDs on another axial level, from bottom to top.
```

Comment:
```suggestion
- [!param](/Mesh/AssemblyMeshGenerator/background_region_ids) is a 1-dimensional array containing region IDs for each axial layer of the background region. duct_region_ids is a 2-dimensional array containing regions IDs for the duct region. The first row of the array represents the 2D duct regions (from innermost to outermost duct region) for the bottom layer of the assembly. Each subsequent row assigns IDs on another axial level, from bottom to top.
```
same for any other param please. it prevents deprecation of the training content

Reply:
Updated, fixed other instances too. Let me know if I missed anything